CMakeLists.txt

```diff
@@ -57,6 +57,17 @@ set(SRCS
 	src/tools/qgraphicspickerscene.cpp
 	src/tools/qundodraw.cpp
 	src/tools/qundoevent.cpp
+    src/commands/changeitem.cpp
+    src/commands/changemoney.cpp
+    src/commands/changeparty.cpp
+    src/commands/changeexperience.cpp
+    src/commands/facegraphics.cpp
+    src/commands/inputnumber.cpp
+    src/commands/messageoptions.cpp
+    src/commands/showchoices.cpp
+    src/commands/showmessage.cpp
+    src/commands/switchoperations.cpp
+    src/commands/variableoperations.cpp
```

Comment:
The problem here is that the Player uses Tabs everywhere but the Editor spaces... Probably because Qt Creator uses Spaces per default... That consistency.
Well, change all to spaces.